weblab/entities/migrations/0016_delete_old_style_fitting_specs.py

```diff
@@ -0,0 +1,19 @@
+# Generated by Django 2.2.16 on 2020-11-25 15:02
+
+from django.db import migrations
+
+
+def delete_protocols(apps):
```

Comment:
It was this function I meant to rename, rather than the whole file. (Not that renaming the file is a problem!)